backend/app/anti_recommendation_engine/anti_recommendation_engine.py

```diff
@@ -39,42 +37,45 @@ def __select_anti_recommender(self) -> AntiRecommender | None:
         """Select and return an AntiRecommender based on values stored in settings."""
 
         if (
-            settings.anti_recommender_type.lower() == "openai"
+            settings.anti_recommender_type == AntiRecommenderType.OPEN_AI
             and settings.openai_api_key
         ):
-            return NormalOpenAiAntiRecommender()
+            return NormalOpenaiAntiRecommender()
+
+        if settings.anti_recommender_type == AntiRecommenderType.ARKG:
+            return ArkgAntiRecommender(
+                arkg_base_iri=settings.arkg_base_iri,
```

Comment:
Don't need all these variable name prefixes.

Reply:
I have removed name prefixes from variables in `ArkgAntiRecommender`.

Reply:
I kept them in `Settings` to distinguish them from similar variables.